js/form.js

```diff
@@ -16,6 +16,11 @@
   var scaleBiggerElement = uploadOverlayElement.querySelector('.scale__control--bigger');
   var scaleInput = uploadOverlayElement.querySelector('.scale__control--value');
   var hashtagsInputElement = uploadOverlayElement.querySelector('.text__hashtags');
+  var commentElement = uploadOverlayElement.querySelector('.text__description');
+
+  var ecsKey = function (event) {
```

Comment:
Переименуй функцию, например
```suggestion
  var ecsKeyHandler = function (event) {
```